.github/workflows/ci-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Build and test
+
+on:
+    # Only on pushes to master or one of the release branches we build on push
+    push:
+        branches:
+            - master
+            - "[0-9].[0-9]+-branch"
```

Comment:
This project does not use *-branch. This line is not needed. It was intended for Pyramid's branching, so I think it just got copy-pasta'd over.

Reply:
I've removed this